### ISSUE TITLE
Cannot connect to SSH host from 32bit windows

### ISSUE THREAD

**Issue by yuntan** [microsoft/vscode-remote-release#178]
When I tried to connect to a SSH host from 32bit windows, I got an error
"The terminal process command 'cmd.exe/c (type "C:\Users\Yuuto\AppData\Local\Temp\vscode-linux-multi-line-command-untan-xyz.sh" | C:\WINDOWS\Sysnative\OpenSSH\ssh.exe  untan-xyz) & exit /b 0' failed to launch (exit code: 255)"

It is because "C:\Windows\Sysnative" is not exist on 32bit windows.

- VSCode Version: 1.34.0-insider(user setup) 32bit
- Local OS Version: Windows 10 Home 1809 32bit
- Remote OS Version: Ubuntu 18.04 64bit
- Remote Extension/Connection Type: SSH

Steps to Reproduce:

1. Execute "Remote-SSH: Connect to Host..." from command pallete

Does this issue occur when you try this locally?: Yes
Does this issue occur when you try this locally and all extensions are disabled?: Yes

**Comment by yuntan** [comment]
Additional info:

```cmd
cmd>where ssh
C:\Program Files\OpenSSH-Win32\ssh.exe
C:\Windows\System32\OpenSSH\ssh.exe
```

**Comment by acedrew** [comment]
Same issue on windows 10 x64

**Comment by garyng** [comment]
Probably duplicate of a https://github.com/microsoft/vscode-remote-release/issues/144?

**Comment by roblourens** [comment]
Yes I'll merge this into #144 thanks

**Comment by yuntan** [comment]
I confirmed that the issue is fixed at Remote - SSH 0.37.0 :+1:

**Comment by roblourens** [comment]
Great! Thanks for confirming.
